src/com/google/enterprise/adaptor/database/DatabaseAdaptor.java

```diff
@@ -406,10 +396,13 @@ public void getDocIds(DocIdPusher pusher) throws IOException,
         ResultSet rs = stmt.executeQuery();
         BufferedPusher outstream = new BufferedPusher(pusher)) {
       log.finer("queried for stream");
+      boolean hasAction
+          = hasColumn(rs.getMetaData(), GsaSpecialColumns.GSA_ACTION);
+      log.log(Level.FINEST, "hasAction: {0}", hasAction);
```

Comment:
Maybe /hasAction/Has gsa_action column/? Are these documented in upper or lower-case?